x/ibc/02-client/keeper/keeper.go

```diff
@@ -40,7 +41,7 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 
 // GetClientState gets a particular client from the store
 func (k Keeper) GetClientState(ctx sdk.Context, clientID string) (exported.ClientState, bool) {
-	store := ctx.KVStore(k.storeKey)
+	store := k.clientStore(ctx, clientID)
 	bz := store.Get(ibctypes.KeyClientState(clientID))
```

Comment:
This seems wrong? `ibctypes.KeyClientState` will prefix the client identifier again, right?

Reply:
(ditto in all the other instances)